config/locales/static/privacy_policy.yml

```diff
@@ -75,14 +77,15 @@ en:
 
         If you opted in to user research when you registered your interest or on the child development training course, you can also contact us to withdraw your consent  at any time. This will not affect your Child development training account, which you will still be able to access as normal.
 
-        To contact us, email [child.development.training@education.gov.uk](mailto://child.development.training@education.gov.uk). 
+        To contact us, email <a class='govuk-link' href='mailto://child.development.training@education.gov.uk'>child.development.training@education.gov.uk</a>.
 
-        If you have any concerns about the way we have handled your personal data, you have the right to raise them with the [Information Commissioner&rsquo;s Office](https://ico.org.uk/concerns){:target="_blank"}.
+        If you have any concerns about the way we have handled your personal data, you have the right to raise them with the <a class='govuk-link' target='_blank' href='https://ico.org.uk/concerns'>Information Commissioner&rsquo;s Office</a>.
         
         ## Last updated
 
         We may need to update this privacy notice periodically, so we recommend that you revisit this information from time to time. This version was last updated on 27 May 2022. 
 
         ## Contact us
 
-        Please [contact us if you have any questions about how your personal information will be used](https://form.education.gov.uk/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-f1453496-7d8a-463f-9f33-1da2ac47ed76/AF-Stage-1e64d4cc-25fb-499a-a8d7-74e98203ac00/definition.json&redirectlink=%2Fen&cancelRedirectLink=%2Fen){:target="_blank"}. Please refer to the &ldquo;Online child development training course&ldquo;.  
+        Please <a class='govuk-link' target='_blank' href='https://form.education.gov.uk/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-f1453496-7d8a-463f-9f33-1da2ac47ed76/AF-Stage-1e64d4cc-25fb-499a-a8d7-74e98203ac00/definition.json&redirectlink=%2Fen&cancelRedirectLink=%2Fen'>contact us if you have any questions about how your personal information will be used</a>.
+        Please refer to the &ldquo;Online child development training course&ldquo;.
```

Comment:
As you swapped the apostrophes in the other file, just checking that these ```&ldquo```s and ```&rsquo;``` were wanted